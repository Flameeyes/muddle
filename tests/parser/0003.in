How about a function?

Hello, I am ${fn:val("/top/my-name-is")} .
