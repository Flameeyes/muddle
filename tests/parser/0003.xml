<?xml version="1.0" ?>

<top>
  <my-name-is>Jim</my-name-is>
</top>
