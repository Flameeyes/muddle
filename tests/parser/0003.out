How about a function?

Hello, I am Jim .
